client/app/scripts/superdesk-archive/module.js

```diff
@@ -288,28 +288,21 @@ define([
                     }],
                     controller: ['data', '$location', 'api', 'notify', 'session', 'desks', 'superdesk',
                         function(data, $location, api, notify, session, desks, superdesk) {
-                            var pick_fields = ['family_id', 'abstract', 'anpa_category',
-                                                'pubstatus', 'slugline', 'urgency', 'subject', 'dateline',
-                                                'priority', 'byline', 'dateline', 'headline'];
-                            var update_item = {};
-                            update_item =  _.pick(angular.extend(update_item, data.item), pick_fields);
-                            update_item.related_to = data.item._id;
-                            update_item.anpa_take_key = 'update';
-                            update_item.task = {};
-
                             session.getIdentity()
                                 .then(function(user) {
-                                    update_item.task.desk = user.desk? user.desk: desks.getCurrentDeskId();
-                                    update_item.state = 'in_progress';
-                                    return api.archive.save({}, update_item);
+                                    var payload = {'desk_id': user.desk? user.desk: desks.getCurrentDeskId()}
```

Comment:
As per [SD-2530](https://dev.sourcefabric.org/browse/SD-2530), the rewritten article should sit on the same desk as the published article.